databend-client/src/main/java/com/databend/client/data/Int8Handler.java

```diff
@@ -543,6 +546,9 @@ private double parseNonNullValue(Object value) {
         if (value.equals("NaN") || value.equals("nan")) {
             return Double.NaN;
         }
+        if (value.equals("Infinity") || value.equals("inf")) {
```

Comment:
Thanks~  
Allow me to suggest a review, the code here is the same as above

Reply:
Thanks！Will fix in next pr.